src/renderer/components/open-project-page/open-project-page.tsx

```diff
@@ -38,6 +46,20 @@ export const OpenProjectPage: FC<{ onNext: Consumer<void>; onBack: MouseEventHan
     }, [error]);
   }
 
+  const useResumeCheck = (onResumeStatus: Consumer<void>) => {
```

Comment:
I think this can be moved out of the component

Reply:
`Consumer<void>` is not a Consumer. Use a different type annotation or name your own.